dockerfiles/rockylinux/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ghcr.io/onlyoffice-qa/rocky-ruby-image:master
```

Comment:
Maybe name the image not the `master` but exact version of ruby it's including

It will simpler to clearly upgrade it later?